includes/class-wc-gateway-stripe.php

```diff
@@ -393,14 +393,6 @@ public function process_payment( $order_id, $retry = true, $force_save_source =
 				$prepared_source = $this->prepare_source( get_current_user_id(), $force_save_source, $stripe_customer_id );
 			}
 
-			// If we are using a saved payment method that is PaymentMethod (pm_) and not a Source (src_) we need to use
-			// the process_payment() from the UPE gateway which uses the PaymentMethods API instead of Sources API.
-			// This happens when using a saved payment method that was added with the UPE gateway.
-			if ( $this->is_using_saved_payment_method() && ! empty( $prepared_source->source ) && substr( $prepared_source->source, 0, 3 ) === 'pm_' ) {
-				$upe_gateway = new WC_Stripe_UPE_Payment_Gateway();
-				return $upe_gateway->process_payment_with_saved_payment_method( $order_id );
-			}
-
```

Comment:
Yup! For clarity (if someone comes to view this PR later) that's because this PR adds support for card payments with Stripe Payment Methods on top of support for Sources 🚀